src/vm.c

```diff
@@ -483,6 +483,26 @@ int tjs__load_file(JSContext *ctx, DynBuf *dbuf, const char *filename) {
     return r;
 }
 
+JSValue TJS_EvalModuleContent(JSContext *ctx, const char *filename, bool is_main, bool use_real_path, const char *content, size_t len) {
```

Comment:
I just moved here to some code from `TJS_EvalModule` to a separate function, because the same code is needed to evaluate the blob inside the ObjectURL as a module. Because the only difference is that we do need to load the content from a file.